public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -24,7 +24,9 @@ function (angular, _, moment, dateMath, kbn, CloudWatchAnnotationQuery) {
 
       var queries = [];
       options = angular.copy(options);
-      options.targets = this.expandTemplateVariable(options.targets, templateSrv);
+      if (!options.scopedVars) {
```

Comment:
not sure this can be correct, I mean the panel could be repeated with one variable, and another normal (non repeated) mult select variable can be used, so just checking for scopedVars is not correct